pkg/services/cosmosdb/common-types.go

```diff
@@ -0,0 +1,20 @@
+package cosmosdb
+
+type databaseKind string
+
+const (
+	databaseKindMongoDB          databaseKind = "MongoDB"
+	databaseKindGlobalDocumentDB databaseKind = "GlobalDocumentDB"
```

Comment:
I think these are a relic of when we used one manager to provision mongo and non-mongo accounts alike.

Now that there are two separate managers, we can hard-code this bit of each information into each manager, which obviates the need for the `databaseKind` type, variables of that type, and tracking that information in the catalog "extras" and the instance details.